platform/android/export/export_plugin.cpp

```diff
@@ -856,8 +857,14 @@ bool EditorExportPlatformAndroid::_has_manage_external_storage_permission(const
 }
 
 bool EditorExportPlatformAndroid::_uses_vulkan() {
-	String current_renderer = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
-	bool uses_vulkan = (current_renderer == "forward_plus" || current_renderer == "mobile") && GLOBAL_GET("rendering/rendering_device/driver.android") == "vulkan";
+	String rendering_method = GLOBAL_GET("rendering/renderer/rendering_method.mobile");
+	String rendering_driver = GLOBAL_GET("rendering/rendering_device/driver.android");
+#ifdef VULKAN_ENABLED
+	if (rendering_driver == "auto" && rendering_method != "gl_compatibility") {
+		rendering_driver = "vulkan";
+	}
+#endif
+	bool uses_vulkan = (rendering_method == "forward_plus" || rendering_method == "mobile") && rendering_driver == "vulkan";
 	return uses_vulkan;
```

Comment:
```suggestion
	return (rendering_method == "forward_plus" || rendering_method == "mobile") && (rendering_driver == "vulkan" || rendering_driver == "auto");
```

Export template can have different set of drivers, so in case of export it's probably better to assume `auto` is a default one for official builds instead of checking `VULKAN_ENABLED`.

Reply:
Since this method adds a `<uses-feature>` entry to the manifest, I think it makes sense to explicitly check whether Vulkan is enabled in the export template.

Reply:
> I think it makes sense to explicitly check whether Vulkan is enabled in the export template.

Yes, but how would we do it? Export plugin is running in the editor, so `VULKAN_ENABLED` only checks if it's enabled in the editor.

Reply:
Yeah... We can't actually check if it's enabled in the template.  

I guess I need to rest now 😅 -- I thought `VULKAN_ENABLED` would check the template.